cdi-langmodel-tck/pom.xml

```diff
@@ -19,9 +19,9 @@
         <maven.compiler.release>21</maven.compiler.release>
         <cdi.tck.version>4.1.0</cdi.tck.version>
         <cdi.api.version>4.1.0</cdi.api.version>
-        <weld.version>6.0.0.Beta1</weld.version>
+        <weld.version>6.0.0.Beta4</weld.version>
         <arquillian.version>1.8.0.Final</arquillian.version>
-        <payara.arquillian.version>3.0</payara.arquillian.version>
+        <payara.arquillian.version>4.0.alpha1</payara.arquillian.version>
```

Comment:
Redundant property - defined in parent pom
```suggestion
```

Reply:
already removed